src/Core/Settings.h

```diff
@@ -221,6 +221,8 @@ class IColumn;
     \
     M(Int64, zstd_window_log_max, 0, "Allows you to select the max window log of ZSTD (it will not be used for MergeTree family)", 0) \
     \
+    M(Bool, enable_case_insensitive_tables, false, "Enables table lookup to be performed regardless of case", 0) \
```

Comment:
Do we need a dedicated setting? The current change extends the list of supported formats, not reduces it.

Reply:
I think that we do need a setting, this change was discussed in #33935 and agreed that changing case for table, databases, and columns should be done independently as settings.